Herebyfile.mjs

```diff
@@ -6,12 +6,14 @@ import { task } from "hereby";
 import _glob from "glob";
 import util from "util";
 import chalk from "chalk";
-import { exec, readJson, getDiffTool, getDirSize, memoize, needsUpdate } from "./scripts/build/utils.mjs";
-import { runConsoleTests, refBaseline, localBaseline, refRwcBaseline, localRwcBaseline } from "./scripts/build/tests.mjs";
+import { exec, readJson, getDiffTool, getDirSize, memoize, needsUpdate, Debouncer, Deferred } from "./scripts/build/utils.mjs";
+import { runConsoleTests, refBaseline, localBaseline, refRwcBaseline, localRwcBaseline, cleanTestDirs } from "./scripts/build/tests.mjs";
 import { buildProject as realBuildProject, cleanProject, watchProject } from "./scripts/build/projects.mjs";
 import { localizationDirectories } from "./scripts/build/localization.mjs";
 import cmdLineOptions from "./scripts/build/options.mjs";
 import esbuild from "esbuild";
+import chokidar from "chokidar";

```

Comment:
Rather than pulling in chokidar here, can we use esbuild's plugin API to detect when `run.js` changes instead? Potentially, we could pass an `EventEmitter` into `entrypointTask`, and then have `runtests-watch` register a handler.

This is actually an API I had intended to add to `hereby`; a callback passed to a task which tells the build system that the task changed, then tasks that depend on it can be rerun. But, it didn't seem like anyone wanted it so I put it off for later.

Reply:
I started with that, but I need to check for more than just run.js.

Reply:
I'm using `chokidar` because its a dependency of `mocha`, so we already have it in our dependency tree.

Reply:
Aha, right! Forgot about that.